docs/serving/services/configure-probing.md

```diff
@@ -0,0 +1,81 @@
+# Configuring Probing
+
+## General understanding of Knative Probing
+
+It is important to note that Knative probing is different from Kubernetes probing. 
+One reason for this is that Knative tries to minimize cold-start time and thus is probing
+in a vastly higher interval than Kubernetes does. 
+
+The general probing architecture looks like this:
+
+![probing-overview](./probes-overview.drawio.svg)
```

Comment:
Might be useful to describe a bit of the details like QP will hit each of the endpoints one by one and how the probe headers are passed. How the re-write looks like?